oneworldcoders/emmanuel_tic-tac-toe
# Basic game play

#### What does this PR do
This PR 
- [x] Adds different files for the classes
- [x] Tests the basic game functionality

#### How can this be tested
* Clone this repo `https://github.com/oneworldcoders/emmanuel_tic-tac-toe.git`
* Follow the README.md to set up the application
* check out this branch `git checkout basic_game_play`
* Run `bundle exec test` to run tests
